app/models/annotation/nml/NmlParser.scala

```diff
@@ -379,8 +380,10 @@ class NmlParser @Inject()(datasetDAO: DatasetDAO) extends LazyLogging with Proto
   private def parseWkUrl(nodes: NodeSeq): Option[String] =
     nodes.headOption.map(node => getSingleAttribute(node, "wkUrl"))
 
-  private def parseOrganizationId(nodes: NodeSeq): String =
-    nodes.headOption.map(node => getSingleAttribute(node, "organization")).getOrElse("")
+  private def parseOrganizationId(nodes: NodeSeq, fallbackOrganization: String): String =
+    nodes.headOption
+      .map(node => getSingleAttributeOpt(node, "organization").getOrElse(fallbackOrganization))
+      .getOrElse(fallbackOrganization)
```

Comment:
in both cases 
- the "experiment" tag does not exist 
- in case the experiment tag has no "organization" field 
the fallback organization (user orga) is used.